eng/scripts/Publish-MavenPackages.ps1

```diff
@@ -307,11 +312,12 @@ foreach ($packageDetail in $packageDetails) {
     else {
       Write-Information "Releasing staging repostiory $stagedRepositoryId"
       Write-Information "mvn org.sonatype.plugins:nexus-staging-maven-plugin:rc-release `"-DstagingRepositoryId=$stagedRepositoryId`" `"-DnexusUrl=https://oss.sonatype.org`" `"-DrepositoryId=target-repo`" `"-DserverId=target-repo`" `"-Drepo.username=$RepositoryUsername`" `"-Drepo.password=`"`"$RepositoryPassword`"`"`" `"--settings=$PSScriptRoot\..\maven.publish.settings.xml`""
-      mvn org.sonatype.plugins:nexus-staging-maven-plugin:rc-release "-DstagingRepositoryId=$stagedRepositoryId" "-DnexusUrl=https://oss.sonatype.org" "-DrepositoryId=target-repo" "-DserverId=target-repo" "-Drepo.username=$RepositoryUsername" "-Drepo.password=""$RepositoryPassword""" "--settings=$PSScriptRoot\..\maven.publish.settings.xml"
+      # TODO: Remove when tested has been completed end-to-end. This final step simply
+      #       closes the staging repo and makes it visible in Maven Central.
+      #mvn org.sonatype.plugins:nexus-staging-maven-plugin:rc-release "-DstagingRepositoryId=$stagedRepositoryId" "-DnexusUrl=https://oss.sonatype.org" "-DrepositoryId=target-repo" "-DserverId=target-repo" "-Drepo.username=$RepositoryUsername" "-Drepo.password=""$RepositoryPassword""" "--settings=$PSScriptRoot\..\maven.publish.settings.xml"
```

Comment:
This will be commented back in once we are 100% happy with everything.

Reply:
Don't forget to uncomment this before merging.

Reply:
Will do. I'm giving an internal team an opportunity to give me an updated set of artifacts for a test run before I open that safety.